src/style/style.css

```diff
@@ -0,0 +1,11 @@
+.rehearsalLinkText {
+    color: #00b380;
```

Comment:
use var:
https://stackoverflow.com/questions/4847850/is-it-possible-to-define-constants-in-css